lake/seekindex/writer.go

```diff
@@ -17,15 +20,15 @@ type Entry struct {
 
 type Writer struct {
 	marshal *zson.MarshalZNGContext
-	writer  zio.Writer
+	writer  zio.WriteCloser
 	offset  uint64
 	valoff  uint64
 }
 
-func NewWriter(w zio.Writer) *Writer {
+func NewWriter(w io.WriteCloser) *Writer {
 	return &Writer{
 		marshal: zson.NewZNGMarshaler(),
-		writer:  w,
+		writer:  zngio.NewWriter(w),
```

Comment:
Tying this package to ZNG doesn't feel quite right. How about changing the io.WriteCloser parameter to zio.WriteCloser and letting the caller create the ZNG writer?

Reply:
Ok